spec/formatting.md

```diff
@@ -18,3 +18,38 @@ the local variable takes precedence.
 
 It is an error for a local variable definition to
 refer to a local variable that's defined after it in the message.
+
+## Bidirectional Isolation
+
+Formatted messages may be bidirectional,
+i.e. contain a mix of left-to-right and right-to-left content.
+In all output formats,
+such parts should be isolated from each other so that
+the directionality of one part of the message does not affect
+the presentation of the rest of the message.
+
+Implementations may provide one or more strategies for isolation,
+but one such strategy must behave as follows,
+when formatting the message as a string:
```

Comment:
I would reverse the normative statements for clarity to implementers:

> Implementations _must_ provide at least one isolation mechanism from the list below. They _may_ provide additional mechanisms in order to allow better overall control from users.

Reply:
At the moment, the PR only presents on mechanism/strategy. If we add the compatibility mode I mention in an earlier comment, I would prefer either requiring both or at least the compatibility mode of all implementations, rather than "at least one".

Reply:
I also think that there's something not right in how this is current phrased. `MAY provide one or more...` and `one MUST behave as follows...` seem to be at odds with each other? Can an implementation provide zero strategies?

I liked the wording suggested by @macchiati in https://github.com/unicode-org/message-format-wg/pull/315#discussion_r1044690668. @aphillips's suggestion above also sounds good to me.